packages/vue3-vite/src/generators/vue3-vite/is-package-based-repo.ts

```diff
@@ -0,0 +1,6 @@
+import { Tree, readJson } from '@nrwl/devkit';
+
+export const isPackageBasedRepo = (tree: Tree): boolean => {
```

Comment:
I didn't manage to find any official solution to tell if a NX repo is package-based or integrated.

From the default NX workspace, it seems that package-based repository would have this `"workspaces": ["packages/*"]` in `package.json`. Is it a reliable way to detect repository type?

By the way, this function is not being used anywhere at the moment, so it's okay to remove it if you think we don't need this!

Reply:
I think it's reasonable! But I'll add a comment so we remember why it's there.